src/stores/Video.js

```diff
@@ -157,6 +157,39 @@ class VideoStore {
     this.rootStore.entryStore.ClearEntries();
   }
 
+  @action.bound
+  GetOfferingKey = flow(function * () {
+    try {
+      const offerings = yield this.rootStore.client.ContentObjectMetadata({
+        libraryId: yield this.rootStore.client.ContentObjectLibraryId({versionHash: this.versionHash}),
+        objectId: this.rootStore.client.utils.DecodeVersionHash(this.versionHash).objectId,
+        metadataSubtree: "offerings",
+        select: [
+          "/*/playout/playout_formats"
+        ]
+      });
+
+      let offering;
+      Object.keys(offerings).some(offeringName => {
+        const playoutFormats = offerings[offeringName].playout.playout_formats;
+        const isHlsClear = Object.values(playoutFormats).some(playoutFormat => playoutFormat.drm === null && playoutFormat.protocol.type === "ProtoHls");
+
+        if(isHlsClear) {
+          offering = offeringName;
+        }
```

Comment:
If there is more than one offering with hls clear, will it be consistent and always return same one? (Server side is written in Go which deliberately returns object keys in random order - I'm not sure if this also applies to JSON returned by server) I recommend `Object.keys(offerings).sort().some(` unless absolutely certain.